grpc-example/grpc-example-scala/build.sbt

```diff
@@ -62,13 +72,16 @@ lazy val `hello-proxy-api` = (project in file("hello-proxy-api"))
 lazy val `hello-proxy-impl` = (project in file("hello-proxy-impl"))
   .enablePlugins(LagomScala)
   .enablePlugins(AkkaGrpcPlugin) // enables source generation for gRPC
+  .enablePlugins(PlayAkkaHttp2Support) // enables serving HTTP/2 and gRPC
   .settings(
   akkaGrpcGeneratedLanguages := Seq(AkkaGrpc.Scala),
     akkaGrpcExtraGenerators += PlayScalaClientCodeGenerator,
   ).settings(
     libraryDependencies ++= Seq(
       lagomScaladslTestKit,
       macwire,
+      playGRPC,
+      akkaHttp2Support,
```

Comment:
Why are `PlayAkkaHttp2Support` and `akkaHttp2Support` required on the `hello-proxy-impl` project? I thought that project only built a gRPC client (in which case I find surprising that we need to enable any HTTP/2 feature).

Hmm, Ok, maybe `akkaHttp2Support` is required to build the gRPC client, but requiring `PlayAkkaHttp2Support` to consume gRPC doesn't look great. We should consider fixing this (as in "don't require `PlayAkkaHttp2Support` to only consume gRPC") in Play itself.